[2021-09-27 15:10:02,210] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.say_hello 2021-09-27T15:00:00+00:00 [queued]>
[2021-09-27 15:10:02,240] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.say_hello 2021-09-27T15:00:00+00:00 [queued]>
[2021-09-27 15:10:02,241] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 15:10:02,242] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-09-27 15:10:02,243] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 15:10:02,256] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): say_hello> on 2021-09-27T15:00:00+00:00
[2021-09-27 15:10:02,261] {standard_task_runner.py:52} INFO - Started process 21262 to run task
[2021-09-27 15:10:02,270] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'say_hello', '2021-09-27T15:00:00+00:00', '--job-id', '71', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmplkcnv0y4', '--error-file', '/tmp/tmpgvr3tyeg']
[2021-09-27 15:10:02,275] {standard_task_runner.py:77} INFO - Job 71: Subtask say_hello
[2021-09-27 15:10:02,337] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.say_hello 2021-09-27T15:00:00+00:00 [running]> on host 7c1d2d2abdb5
[2021-09-27 15:10:02,467] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Veerawat
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=say_hello
AIRFLOW_CTX_EXECUTION_DATE=2021-09-27T15:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-27T15:00:00+00:00
[2021-09-27 15:10:02,469] {my_dag.py:11} INFO - Hello, SAKSIAM
[2021-09-27 15:10:02,471] {python.py:151} INFO - Done. Returned value was: None
[2021-09-27 15:10:02,493] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=say_hello, execution_date=20210927T150000, start_date=20210927T151002, end_date=20210927T151002
[2021-09-27 15:10:02,563] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-09-27 15:10:02,613] {local_task_job.py:261} INFO - 1 downstream tasks scheduled from follow-on schedule check
